Resources/Private/Partials/Organisationseinheiten/AnschriftUndKommunikation.html

```diff
@@ -17,6 +17,20 @@
     </f:if>
   </dd>
 </dl>
+
+<f:if condition="{internetadressen}">
+  <dl class="extdl clearfix">
+    <dt><f:translate key="organisationseinheit.internetadressen" /></dt>
+    <f:for each="{internetadressen}" as="internetadresse">
+      <dd>
+        <span aria-hidden="true" class="icon-earth"></span>
+        <a href="{internetadresse.uri}" title="{internetadresse.titel}"{f:if(condition: '{internetadresse.kennzeichenAnzeigeNeuesFenster}', then: ' target="_blank"', else: '')}>
```

Comment:
There is no need to wrap a variable in '{ and }'. It's only needed, if you check against a Fluid VH.